Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs

```diff
@@ -102,6 +102,34 @@ internal partial class PI
         internal static readonly IntPtr HWND_BOTTOM = new IntPtr(1);
 
         internal const int BM_CLICK = 0x00F5;
+
+        // Menu item info mask constants
+        internal const uint MIIM_STATE = 0x00000001;
```

Comment:
Why this pattern and not the class pattern used by the other enum definitions

Reply:
Fixed